packages/ove-app-audio/src/client/players/howler.js

```diff
@@ -50,26 +52,28 @@ function OVEHowler () {
         this.player.mute(false);
     };
 
-    this.setVolume = function(volume) {
+    this.setVolume = function (volume) {
         log.debug('setting volume to');
-        this.player.volume(volume);
+        this.player.volume(parseFloat(volume));
```

Comment:
What happens if I set volume to less than 0 or greater than 1?

Reply:
there is internal howler validation for this. to avoid code bloat i didn't see sense in re-implementing these checks.